MicroProfile-Metrics/tck-runner/pom.xml

```diff
@@ -157,6 +176,41 @@
                 <cdi.version>2.0</cdi.version>
             </properties>
         </profile>
+        <profile>
+            <id>mp-metrics-2.2</id>
+            <activation>
+                <property>
+                    <name>payara.version</name>
+                    <value>/5\.194.*|5\.201.*/</value>
+                </property>
+            </activation>
+            <properties>
+                <!-- Metrics Dependencies -->
+                <microprofile.metrics.version>2.2.payara-p1</microprofile.metrics.version>
+                <microprofile.metrics.tck.version>2.2.payara-p1</microprofile.metrics.tck.version>
+
+                <!-- Other Test Dependencies -->
+                <cdi.version>2.0</cdi.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>mp-metrics-2.3</id>
+            <activation>
+                <property>
+                    <name>payara.version</name>
+                    <value>5\.20[^1].*/</value>
```

Comment:
This doesn't match against 5.202-SNAPSHOT either, but gets by due to default property value.
It seems to outright fail though against 5.202.1-SNAPSHOT - weirdly because it seems to have lost the default value:

```
andrew@OpenSUSE:~/Git/MicroProfile-TCK-Runners> mvn help:active-profiles  -Dpayara.version=5.202.1-SNAPSHOT -Ppayara-server-remote -pl MicroProfile-Metrics/tck-runner
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.eclipse.microprofile.rest.client:microprofile-rest-client-tck:jar must be a valid version but is '${microprofile.rest.client.tck.version}'. @ line 105, column 22
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
```

Reply:
I think it is because `5\.20[^1].*/` is missing the `/` at the beginning.